docs/.vuepress/components/StartBuildingPage.vue

```diff
@@ -0,0 +1,255 @@
+<template>
+  <div class="start-building">
+    <div class="center">
+      <div class="headline-text-container">
+        <h1 class="headline-title">Get started building on Ethereum</h1>
+        <div class="headline-subtitle">Learn the basics of Ethereum with Ethereum Studio, our web-based IDE where you can create and test smart contracts and their front end.</div>
```

Comment:
```suggestion
        <div class="headline-subtitle">Learn the basics of Ethereum with Ethereum Studio, our web-based IDE where you can create and test smart contracts, and build a front end for them.</div>
```

Something like this, it felt like we were saying "test the front end"